cookbooks/cdo-varnish/test/shared/shared.rb

```diff
@@ -489,13 +489,15 @@ def does_not_strip_cookies_from_png_in_path(path)
         end
       end
 
-      it 'Strips cookies from the penultimate dance level' do
-        assert strips_session_specific_cookies_from_request? '/s/dance/lessons/1/levels/12'
-        assert strips_session_specific_cookies_from_request? '/s/dance-2019/lessons/1/levels/9'
+      it 'Strips cookies from dance levels' do
```

Comment:
I was unable to verify these tests locally and I don't think Drone runs them either so DTT may be the first place these will be run.